### PR TITLE
format the arrangement of the license terms

### PR DESCRIPTION
**Pull request by Fienne**

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -28,16 +28,14 @@ Under the following terms:
...
 anything the license permits.  With the understanding that:
 
-Notices:
-
-* You do not have to comply with the license for elements of the
+  * You do not have to comply with the license for elements of the
   material in the public domain or where your use is permitted by an
   applicable exception or limitation.
-* No warranties are given. The license may not give you all of the
+  * No warranties are given. The license may not give you all of the
```
Comment (kinow) [inline LICENSE.md:38]: These two list items appeared to be rendered correctly before. Or were they broken @Fienne ?

image

https://github.com/common-workflow-language/user_guide/blob/f9e63daf90cb84fb23e5df678592528b451263e0/LICENSE.md

Comment (Fienne) [reply LICENSE.md:38]: They were rendered correctly. I thought they are sub lists to the No additional restrictions list and had to be arranged in a way that is more readable like so: 
Screenshot from 2022-10-18 23-00-27 1

Comment (kinow) [reply LICENSE.md:38]: Ah! Now I see. They were not related to the paragraph above with "Notices:"… I think. It was really confusing, great catch. I think your version is correct, but leaving it up to @tetron or @mr-c or @swzCuroverse to confirm. Thanks for clarifying!

**Review by kinow — APPROVED** [review]

File: LICENSE.md (hunk 1 above)
Comment (Fienne) [reply LICENSE.md:38]: Thank you :)

**Review by tetron — APPROVED** [review]

**Pull request merged**